test/e2e/resize_nodes.go

```diff
@@ -361,10 +361,15 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 		}
 	}()
 
+	Logf("Waiting for node %s to be ready", node.Name)
```

Comment:
Sorry if this is a silly question, but why are you waiting for the node to be ready in this function, when it is also checked right after the function? Don't we want to isolate the failure from the checking?